distributed_index_tests/lib/user.py

```
@@ -117,22 +111,16 @@ def get_num_docs(self, doc_name_pattern):
                 docs.append(obj['id'])
         return len(docs)
 
-
-    def admin_add_user(self, user):
-        headers = {"Content-Type": "application/json"}
-        self.admin_db_url = "{0}/{1}".format(self.admin_url, self.db)
-        json_doc = json.dumps({"name": self.name, "password": self.password, "admin_channels": self.channels})
-        response = requests.put("{0}/_user/{1}".format(self.admin_db_url, self.name), headers=headers, data=json_doc)
-        if response.status_code == 201:
-            print "User:",user,"added successfully"
-            return True
+    def get_changes(self, with_docs=False):
+        if with_docs:
+            r = requests.get("{}/{}/_changes?include_docs=true".format(self._url, self.db), headers=self._headers)
         else:
-            print "Error: Could not add User",user,response.json()
-            return False
+            r = requests.get("{}/{}/_changes".format(self._url, self.db), headers=self._headers)
+        return json.loads(r.text)
```

Please throw an exception on non expected response codes.